src/pymorize/units.py

```diff
@@ -0,0 +1,108 @@
+import pint
+import re
+
+
+"""
+This module deals with the auto-unit conversion in the cmorization process.
+In case the units in model files differ from CMIP Tables, this module attempts to convert them automatically.
+
+In case of missing units in either model files or CMIP Tables, this module can not convert from a dimentionless base to something with dimension. Dealing with such thing have to done with `action` section in the Rules module on a per variable basis.
+
+"""
+
+
+# TODO: decide a place to hold the missing unit definitions that pint is not aware of
+# from pathlib import Path
+# _unitsfile = Path(__file__).parent / "units_en.txt"
+# u = pint.UnitRegistry(_unitsfile)
```

Comment:
Either remove this or use it correctly. From my side, I would rather keep everything unit related in one file for simplicity.

```suggestion
```